src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -29,27 +28,16 @@ const AnonPostNotificationChannelItem: (props: MessageChannelItemProps) => React
   let level1FirstComment: Reaction = null;
   let level2FirstComment: Reaction = null;
 
-  const helperDeterminePostType = (
-    isOwnPost: boolean,
-    isOwningReaction: boolean
-  ): BaseChannelItemTypeProps => {
-    let type = BaseChannelItemTypeProps.ANON_POST_NOTIFICATION;
-    if (isOwnPost && isOwningReaction)
-      type = BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_I_COMMENTED;
-    else if (isOwnPost) type = BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION;
-    else if (isOwningReaction) type = BaseChannelItemTypeProps.ANON_POST_NOTIFICATION_I_COMMENTED;
-
-    return type;
-  };
-
   const helperDetermineCommenterName = () => {
     const anonymousCommenterName = firstComment?.reaction?.data?.anon_user_info_emoji_name;
     const firstCommenterId = firstComment?.reaction?.user_id;
     const isFirstCommenterMe =
       firstCommenterId === signedProfileId || firstCommenterId === anonProfileId;
 
-    if (firstComment?.reaction?.isOwningReaction) commenterName = 'You';
-    else if (anonymousCommenterName) {
+    if (firstComment?.reaction?.isOwningReaction) {
+      commenterName = 'You';
+      postNotificationPicture = firstComment?.reaction?.user?.data?.profile_pic_url;
+    } else if (anonymousCommenterName) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The helper function `helperDetermineCommenterName` has been modified. The logic seems to be the same, but the assignment of `postNotificationPicture` has been moved inside the if condition. This change could potentially affect the rendering of the component, so make sure it's thoroughly tested.

```diff
- if (firstComment?.reaction?.isOwningReaction) commenterName = 'You';
+ if (firstComment?.reaction?.isOwningReaction) {
+   commenterName = 'You';
+   postNotificationPicture = firstComment?.reaction?.user?.data?.profile_pic_url;
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->